js/src/components/SnippetPreviewSection.js

```diff
@@ -17,6 +17,22 @@ const Section = styled( StyledSection )`
 	}
 `;
 
+/**
+ * Function to map the editor data to data for the preview.
+ *
+ * @param {Object} mappedData The default mappedData object.
+ * @param {string} mappedData.title The processed title.
```

Comment:
What does 'processed' mean in "The processed title"?  Is it the same as mapped? Also: does this function/the programmer using this function need to know that the title has been processed/mapped before? And: does the title even have to be processed/mapped before? (see previous comment)

Reply:
In this case the processed means that some replacement variables in the originalData.title have been replaced, which happens in `yoast-components`. I think it is not that important to know that it has been done before. This function is to do it again for the specifics of the plugin.